SCHEMA.md

```diff
@@ -7,15 +7,37 @@ This schema works across any EAS-supported blockchain network, making it future-
 
 ### Schema Details
 
-```json
+```javascript
 {
-    "roundIds"          : "(uint64,uint64)[]", // array of tuple(chainId, roundId)
-    "totalUSDAmount"    : "uint64", // total contribution
-    "timestamp"         : "uint64", // time of donation
-    "metadataPtr"       : "string", // Link to details about contribution (eg: IPFS)
+    "projectsContributed"   : "uint64" // networks where the round reside
+    "roundsContributed"     : "uint64", // rounds contributed to
+    "chainIdsContributed"   : "uint64", // chainIds contributed to
+    "totalUSDAmount"        : "uint128", // total contribution
```

Comment:
- should this be uint64 ?
- Also how do we prevent duplicate attestations for the same cart ?

Reply:
yes, uint64

we cant prevent duplicate attestations. maybe we really should consider to attest always the whole history.  basically continuously update the attestation @thelostone-mc